### ISSUE TITLE
Not getting predited_probability for counterfactuals using RandomForest Classifier

### ISSUE THREAD

**Issue by vdesai24** [interpretml/DiCE#232]
Hi,

I am trying to generate counterfactuals for a classification model. Trained the model using RandomForestClassifier. 
I am expecting the target variable should show the predicted probability. It is showing predicted class instead.
What changes do I need to make  to give me the predcited probabilities in the generated counterfactuals.
Below is the code to generate counterfactuals:
model = RandomForestClassifier()
d = dice_ml.Data(dataframe=train_dataset, continuous_features=['age', 'hours_per_week'], outcome_name='income')
m = dice_ml.Model(model=model, model_type='classifier',backend="sklearn")
exp = dice_ml.Dice(d, m, method="random")

e1 = exp.generate_counterfactuals(x_test[0:1], total_CFs=5, desired_class='opposite')
e1.visualize_as_dataframe(show_only_changes=True)

Output:
![image](https://user-images.githubusercontent.com/48753044/133968320-77b28024-0e5c-4e1e-b5b4-2df17f10dc6d.png)
![image](https://user-images.githubusercontent.com/48753044/133968353-66a3a6bb-8b2f-4547-a547-2491fb29d047.png)

I need predicted probabilities instead of the predicted class in the outcome.

**Comment by gaugup** [comment]
@vdesai24, the output for the counterfactual is currently only shows the desired_class which is either 0 or 1. We could add another column which can contain the probabilities of the target class in classification scenario. 

@amit-sharma what do you think about adding additional column for adding probabilities in classification scenarios?

Regards,

**Comment by amit-sharma** [comment]
yes, that's a good idea. We just need to call the classifier again on the returned CF data to obtain the predicted probabilities. This would be a general solution that can work for any CF method. For efficiency, we may make it as an optional flag, so that this additional computation is not done.

**Comment by amit-sharma** [comment]
We will have to see how to prioritize this change, given the other enhancements we are working on. Hopefully this will be included in the next release, near Dec.

**Comment by vdesai24** [comment]
Thanks @amit-sharma

**Comment by LingAdeu** [comment]
Hi @amit-sharma may I know if this enhancement has been updated? Thanks!
